Gemfile

```diff
@@ -2,7 +2,7 @@ source "https://rubygems.org"
 
 ruby file: ".ruby-version"
 
-gem "rails", "~> 7.1.5.2"
+gem "rails", "~> 7.1.6"
```

Comment:
Maybe we could also take this chance to update the publiccode.yml file? 🤔 
Do you think we should also mention it in the wiki section [Upgrading to a new version of Rails](https://github.com/consuldemocracy/consuldemocracy/wiki/Upgrading-to-a-new-version-of-Rails)?

Reply:
Definitely worth adding it to the wiki :+1:. The steps in the wiki are only meant for major/minor versions of Rails, though, and not for patch releases :thinking:, so I guess we'd still miss it in cases like this one :sweat_smile:.